Cassiopee/Generator/Generator/IBM.py

```diff
@@ -1120,7 +1132,7 @@ def createRefinementBodies(tb, dimPb=3, hmod=0.01):
     import Geom.IBM as D_IBM
     import Geom.Offset as O
 
-    pointsPerUnitLength = 10 if dimPb == 3 else 100
+    pointsPerUnitLength = 25 if dimPb == 3 else 1000
```

Comment:
Could these eventually be made into user input variables with default values? It has been shown that the previously chosen good parameters have evolved as more test cases are tested.

Reply:
Ah, sorry, you told me that last time and I forgot to change that bit! I modified the code in another commit.